src/js/tabs/pid_tuning.js

```diff
@@ -32,8 +32,9 @@ TABS.pid_tuning.initialize = function (callback) {
         self.activeSubtab = 'pid';
     }
 
-    // Update filtering defaults based on API version
+    // Update filtering and pid defaults based on API version
     const FILTER_DEFAULT = FC.getFilterDefaults();
+    const PID_DEFAULT = FC.getPidDefaults();
```

Comment:
Need to check why it was not needed before

Reply:
Because we never haven't used fc.js so far to interact with PID defaults here, time to do this with all defaults in a future PR.